src/Fixer/ClassNotation/FinalPublicMethodForAbstractClassFixer.php

```diff
@@ -64,7 +64,7 @@ public function start()
                 ),
             ],
             'Enforce API encapsulation in an inheritance architecture. '
-            .'If you want to override a method, use the Template method pattern.',
+                .'If you want to override a method, use the Template method pattern.',
```

Comment:
```
            'Enforce API encapsulation in an inheritance architecture. '
                .'If you want to override a method, use the Template method pattern.',
            'Risky when overriding `public` methods of `abstract` classes.'
```
this added indent on the middle line is intended?

Reply:
It is, it adds intent before the dot. The line `'Risky when overriding...` is another argument.

Reply:
Thanks for the details. I'm not 100% sure I follow the rational for the change the rule makes all the time, but that should not be of concern by itself per se.
I do wonder, as the rule is part of PER-CS2; do we find the rule with the change as proposed here in the PER-CS2 standard, or should this new behavior be opt-in through configuration?

Reply:
If you're interested in the history @SpacePossum

I was working on a FullyMultilineFixer for PER-CS2 and got an issue https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7813/files#r1531647667

```
if ($a
    || $b) {
    // foo
}
```
was changed to
```
if (
$a
    || $b) {
    // foo
}
```
but then the StatementIndentationFixer change the code to
```
if (
    $a
        || $b) {
    // foo
}
```
this is explained here https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7884#issue-2185751932

So I understand correctly the implementation made by the original author in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7901, it requires that the StatementIndentationFixer does not rely anymore on the existing indentation, but have an (opinionated ?) rule for indentation and inner-indentation.

But maybe indeed, some config could be added for the inner-indentation. (It can also be done in another PR, the iteration seems doable).